packages/shared/providers.yaml

```diff
@@ -505,6 +513,8 @@ auth0:
     refresh_params:
         grant_type: refresh_token
     docs: https://docs.nango.dev/integrations/all/auth0
+    proxy:
+        base_url: https://${connectionConfig.subdomain}
```

Comment:
`https://${connectionConfig.subdomain}.auth0.com`?

Reply:
according to the docs its `https://${connectionConfig.subdomain}`  😅 
<img width="551" alt="Screenshot 2024-12-12 at 00 15 34" src="https://github.com/user-attachments/assets/f86034c3-49d5-4c2d-9038-83c438607ea3" />
They share the same with the authorize and token urls

Reply:
but we put `https://${connectionConfig.subdomain}.auth0.com` in the token_url

Reply:
Makes sense, to have the suffix `auth0.com`, I have updated with the latest commit.